### ISSUE TITLE
Example 3, type of refs in <orth> elements

### ISSUE THREAD

**Issue by pdurusau** [digitaltolkien/a-middle-english-vocabulary#4]
The references (XII) and (XV) are left as XML comments but shouldn't they be:

&lt;orth&gt;Briht &lt;oRef&gt;XII&lt;/oRef&gt;&lt;/orth&gt;
&lt;orth&gt;Bryht &lt;oRef&gt;XV&lt;/oRef&gt;&lt;/orth&gt;

&lt;oRef &gt;inherits all the pointer attributes but the TEI presumes pointing towards a headword? Or at least that is how I read the Guidelines. 

Sadly all &lt;orth&gt; can contain is &lt;oRef&gt; and &lt;g&gt; gaiji (character data)

Should we supplement this references? XII, Birht, occurs once on line 130; XV, Bryth, occurs once on line 26. 

Q: Should you use &lt;oRef&gt; @type "occurrence" plus a pointer into the text? Q: But then how to distinguish author's reference from our more precise reference? 

Updated to use character entities, most of what I was saying was lost without them.

**Comment by jtauber** [comment]
They are in comments because I hadn't worked out how to convey them in XML yet.

They are obviously just indicating which spelling in used in which work. I hadn't yet worked out how to best express this. The closest I could find in the TEI P5 guidelines is something like the `<usg>` tag.

**Comment by pdurusau** [comment]
I think of &lt;usg&gt; , https://www.tei-c.org/release/doc/tei-p5-doc/en/html/DI.html#DITPUS, as being more about the sense of a word than its location in a particular text. They could be separated by &lt;form&gt; and the separate &lt;form&gt; elements linked to occurrences in the text with out of line linking.

**Comment by jtauber** [comment]
Yeah `<usg>` definitely isn't quite right but it's the only example in the P5 guidelines of adding any kind of scoping to an `<orth>` (rather than, say, a sense). I presume it's used for US vs UK spelling (for example) which is a fairly similar use case.
